app/controllers/items/new.js

```diff
@@ -69,6 +70,23 @@ export default GoodcityController.extend(
       return this.get("i18n").t(str);
     },
 
+    allAvailablePrinter: Ember.computed(function() {
+      return this.get("printerService").allAvailablePrinter();
+    }),
+
+    selectedPrinterDisplay: Ember.computed({
+      get(key) {
+        let userPrinter = this.get("session").userDefaultPrinter();
+        if (!userPrinter) {
+          return [];
```

Comment:
@akshaygwd , I think, in case of no default printer it should send the `firstObject` from the printers for that location as discussed in the call once. Because sending a blank array will display nothing on the printer selection dropdown.